modules/apps/29-fee/keeper/relay_test.go

```diff
@@ -0,0 +1,57 @@
+package keeper_test
+
+import (
+	clienttypes "github.com/cosmos/ibc-go/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+)
+
+func (suite *KeeperTestSuite) TestWriteAcknowledgementAsync() {
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success",
```

Comment:
maybe add a test case for the forward relayer address being set?

Reply:
Done. Added a check to ensure it gets successfully deleted if set, and no-ops otherwise.